docs/ovms_cloud_native_quickstart.md

```diff
@@ -0,0 +1,298 @@
+
+> **Note**: - OpenVINO Model Server can run on Linux and macOS. For use on Windows, [WSL](https://docs.microsoft.com/en-us/windows/wsl/) is required.
+
+To quickly start using Cloud Native OpenVINO™ Model Server follow these steps:
+1. Install Kubernetes Cluster
+2. Install OpenVINO Toolkit Operator
+3. Provide a model in PV/PVC
+4. Deploy an OpenVINO Model Server Instance
+5. Verify OpenVINO Model Serving by sending Inference Request
+
+### Step 1: Install Kubernetes Cluster
+
+There are many methods to install a Kubernetes Cluster, such as `kind`, `minikube` and `kubeadm`. This guide is using [Creating a cluster with kubeadm](https://kubernetes.io/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm/) to create a Kubernetes cluster.
+
+### Step 2: Install OpenVINO Toolkit Operator
+
+This Kubernetes operator named [OpenVINO Toolkit Operator](https://operatorhub.io/operator/ovms-operator) manages OpenVINO components in Kubernetes, and currently available components are ModelServer. Please follow instructions:
+
+- 1. Install Operator Lifecycle Manager (OLM) on your cluser:
+
+```bash
+$ curl -sL https://github.com/operator-framework/operator-lifecycle-manager/releases/download/v0.27.0/install.sh | bash -s v0.27.0
+```
+
+- 2. Verify the OLM on your cluster:
+
+```bash
+$ kubectl get pods -n olm
+NAME                                READY   STATUS    RESTARTS   AGE
+catalog-operator-74944d49c6-68257   1/1     Running   0          5m
+olm-operator-5c789d6974-4w2sz       1/1     Running   0          5m
+operatorhubio-catalog-jwmdp         1/1     Running   0          5m
+packageserver-6d5b65f789-24dql      1/1     Running   0          5m
+packageserver-6d5b65f789-7bbk5      1/1     Running   0          5m
+```
+
+- 3. Install the operator by running the following command:
+
+```bash
+$ kubectl create -f https://operatorhub.io/install/ovms-operator.yaml
+```
+
+- 4. Verify the operator by using below command:
+
+```bash
+$ kubectl get csv -n operators
+NAME                       DISPLAY                     VERSION   REPLACES                   PHASE
+openvino-operator.v1.1.0   OpenVINO Toolkit Operator   1.1.0     openvino-operator.v1.0.0   Succeeded
+
+$ kubectl get pods -n operators
+NAME                                                    READY   STATUS    RESTARTS   AGE
+openvino-operator-controller-manager-548888ddc7-jq4mh   2/2     Running   0          5m
+```
+
+### Step 3: Provide a model in PV/PVC
+
+As an end-to-end example, this guide uses `PV/PVC` to store the AI model, and user can also store their model by using [Cloud Storage](using_cloud_storage.md). Before the storage of AI model, please refer to [Model Repository](models_repository.md) to know how to organize components of the model. 
```

Comment:
Please change these links to absolute ones from the model_server repository (https://github.com/openvinotoolkit/model_server/tree/main/docs):

 using_cloud_storage.md
 models_repository.md
 starting_server.md
 model_version_policy.md
 parameters.md
 features.md
 ../demos

Reply:
Thanks @pgladkows for this comment! Done!